utils/helpers.js

```diff
@@ -0,0 +1,21 @@
+export function setHeaders(
+ res, headers = {}
+) {
+
+for( const name in headers ) {
+
+    if( ! headers[ name ] )
```

Comment:
Why are you deleting `headers[ name ]`?

Why can't just setHeader? If there's already one, it's gonna update it, otherwise it's gonna set a new one.